src/app/likes/likes.component.css

```
@@ -0,0 +1,16 @@
+:host {
+  align-items: center;
+  display: flex;
+  height: 95%;
+  justify-content: center;
+  color: #717f77;
```

style nit: alphabetize these properties.

any time you have a list of stuff, it's good to keep them alphabetized - it makes them easy to scan and helps avoid merge conflicts where two people add a value to the end of the same list.

use this as an opportunity to set up an editor plugin/shortcut for this (vim has `:sort`)

```suggestion
  align-items: center;
  color: #717f77;
  display: flex;
  height: 95%;
  justify-content: center;
```